php/Dockerfile

```diff
@@ -70,8 +70,8 @@ RUN composer global require hirak/prestissimo --prefer-dist --no-progress --no-s
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
   apt-get install -y nodejs
 
-# Install Grunt CLI
-RUN npm install -g grunt-cli
+# Install Grunt CLI & Yarn
+RUN npm install -g grunt-cli yarn
```

Comment:
Why using npm to install Yarn? x)

Reply:
@maxhelias juste add it to the apt-get install list!

Reply:
Why is yarn needed ? npm isn't enough ?

Reply:
@matthieupetel, because it seems to be faster than npm. Here is a recent [benchmark](https://github.com/appleboy/npm-vs-yarn).